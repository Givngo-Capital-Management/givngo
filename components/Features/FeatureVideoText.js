import React from 'react';
import styles from './FeatureVideoText.module.scss';
import FeatureVideo from './FeatureVideo';

export default function FeatureaVideoText() {
  return (
    <div className={styles['video-text-section']}>
      <div className={styles['video-text-wrapper']}>
        <div className={styles['text-wrapper']}>
          <h1 className={styles.heading}>Videos that engage</h1>
          <p className={styles.paragraph}>
            The most effective way to get people engaged and sharing online is
            through a great story with a great video. This is why givngo focuses
            on helping members create videos that tell their story in a way that
            creates action. Our in house team of video creators and success
            managers are always ready to givngo!
          </p>
        </div>
        <div className={styles['video-wrapper']}>
          <FeatureVideo
            id="2F371994595"
            title="Save the Children, Who We Are"
          />
        </div>
      </div>
    </div>
  );
}
