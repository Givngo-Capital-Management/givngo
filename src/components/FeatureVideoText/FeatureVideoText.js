import React from 'react';
import './FeatureVideoText.scss'
import FeatureVideo from './FeatureVideo/FeatureVideo';

export default function FeatureaVideoText() {
  return (
    <div className="video-text-section">
      <div className="video-text-wrapper">
        <div className="text-wrapper">
          <h1 className="heading">Videos that engage</h1>
          <p className="paragraph">The most effective way to get people engaged and sharing online is through a great story with a great video. This is why givngo focuses on helping members create videos that tell their story in a way that creates action. Our in house team of video creators and success managers are always ready to givngo!</p>
        </div>
        <div className="video-wrapper">
          <FeatureVideo className="video" id="2F371994595" title="Save the Children, Who We Are"/>
        </div>
      </div>
    </div>
  )
}
