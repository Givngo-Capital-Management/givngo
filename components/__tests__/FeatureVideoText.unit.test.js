import React from 'react';
import { shallow } from 'enzyme';
import FeatureVideoText from '../Features/FeatureVideoText';

describe('FeatureVideoText', () => {
  test('renders', () => {
    const wrapper = shallow(<FeatureVideoText />);
    expect(wrapper).toMatchSnapshot();
  });
});
