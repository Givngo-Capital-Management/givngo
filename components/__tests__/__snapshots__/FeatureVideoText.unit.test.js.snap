// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureVideoText renders 1`] = `
<div>
  <div>
    <div>
      <h1>
        Videos that engage
      </h1>
      <p>
        The most effective way to get people engaged and sharing online is through a great story with a great video. This is why givngo focuses on helping members create videos that tell their story in a way that creates action. Our in house team of video creators and success managers are always ready to givngo!
      </p>
    </div>
    <div>
      <FeatureVideo
        id="2F371994595"
        title="Save the Children, Who We Are"
      />
    </div>
  </div>
</div>
`;
